mapdata/WarcraftLegacies/AbilityData/Core/1919378753.json

```diff
@@ -9,14 +9,7 @@
       "Pointer": 1,
       "Id": 828794180,
       "Type": 2,
-      "Value": 0.7
-    },
-    {
-      "Level": 1,
-      "Pointer": 5,
-      "Id": 895903044,
-      "Type": 2,
-      "Value": 0.7
+      "Value": 0.85
```

Comment:
Just so I don't have to run your branch in-game for this one.... Could you please confirm that the Elune's Grace change did not need a tooltip update, i.e. that it uses data fields?

Reply:
Correct, elune's grace already has data fields